community/iftop/build

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -e
+
+export CFLAGS="$CFLAGS -fcommon"
+
+./configure \
+	--prefix=/usr \
+	--sbindir=/usr/bin \
+	--mandir=/usr/share/man
```

Comment:
https://kisscommunity.bvnf.space/kiss/style-guide/#0202

Reply:
owww, i was using spaces in my code editor but packaging for void-packages made me switch fully to using tabs